.coafile

```diff
@@ -39,7 +39,7 @@ max_lines_per_file = 1000
 
 [rst]
 bear = reSTCheckBear
-files = README.rst
+files = **/*.rst
```

Comment:
Seems ok... Here is my coala run output(I think the double star also matches none)
````
Executing section rst...
[DEBUG][19:14:03] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][19:14:03] Files that will be checked:
/home/dev/projects/dev/coala-bears/RELEASE_NOTES.rst
/home/dev/projects/dev/coala-bears/README.rst
/home/dev/projects/dev/coala-bears/docs/index.rst
````